src/System.Private.CoreLib/shared/System/Number.Parsing.cs

```diff
@@ -32,6 +32,15 @@ internal partial class Number
         private const int Int64Precision = 19;
         private const int UInt64Precision = 20;
 
+        private const int DoubleMaxExponent = 309;
+        private const int DoubleMinExponent = -324;
+
+        private const int FloatingPointMaxExponent = DoubleMaxExponent;
+        private const int FloatingPointMinExponent = DoubleMinExponent;
```

Comment:
Why do we need these rather than just using DoubleMax/MinExponent?

Reply:
It makes it easier to expand to support things like `Binary128`, if that ever happens.

Reply:
Ok.